graph/src/components/subgraph/proof_of_indexing/mod.rs

```diff
@@ -76,14 +76,17 @@ mod tests {
                 break;
             }
 
+            let block_number = (block_count - 1) as u64;
+            let block_ptr = BlockPtr::from((case.data.block_hash, block_number));
+
             for block_i in 0..block_count {
                 let mut stream = ProofOfIndexing::new(block_i.try_into().unwrap(), version);
 
                 for (name, region) in case.data.causality_regions.iter() {
                     let block = &region.blocks[block_i];
 
                     for evt in block.events.iter() {
-                        stream.write(&logger, name, evt);
+                        stream.write(&logger, name, evt, block_ptr.clone());
```

Comment:
block_ptr passed here does not match the block for which the POI is for. 

Looks like getting the block_hash here is tricky.

Reply:
changed the code to only log `block_number`